docs/changelog.md

```diff
@@ -4,6 +4,12 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v2.3.0 - Apr 27, 2022
+### Bug Fixes
+
+### Major Features and Improvements
+* Added bolt over `WebSocket` support to Memgraph server. [#384](https://github.com/memgraph/memgraph/pull/384)
```

Comment:
```suggestion
* Added Bolt over `WebSocket` support to Memgraph. [#384](https://github.com/memgraph/memgraph/pull/384)
```

I think we use Bolt with capital B.
`Memgraph server` is not clear in my opinion. We have Memgraph (which might refer to both community version and Memgraph Enterprise) and we have Memgraph monitoring server. Am I missing something here?

Reply:
The server is here just in the context of client-server communication, since it's not a `Server` it's neither a feature nor an additional product. But I will remove it if it's confusing